manifests/d/drakkan/SFTPGo/v2.4.2/drakkan.SFTPGo.locale.en-US.yaml

```
@@ -0,0 +1,30 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: drakkan.SFTPGo
+PackageVersion: v2.4.2
+PackageLocale: en-US
+Publisher: Nicola Murino
+PackageName: SFTPGo
+PackageUrl: https://github.com/drakkan/sftpgo
+License: AGPL-3.0
+Copyright: AGPL-3.0
```

```suggestion
Copyright: Copyright (c) Drakkan
```
This is a license, not a copyright. Please update this to the suggestion or your desired copyright.